kbase-extension/static/kbase/js/common/ui.js

```diff
@@ -34,11 +34,11 @@ define([
 
     function htmlEncode(str) {
         return str
-            .replace(/&/, '&amp;')
-            .replace(/'/, '&#039;')
-            .replace(/"/, '&quot;')
-            .replace(/</, '&lt;')
-            .replace(/>/, '&gt;');
+            .replace(/&/g, '&amp;')
```

Comment:
I assume that these encode functions are supposed to encode every symbol in a string, not just the first one...